src/test/java/UserIdGeneratorTest.java

```diff
@@ -1,2 +1,40 @@
-package PACKAGE_NAME;public class UserIdGeneratorTest {
+import models.*;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.lang.reflect.Field;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserIdGeneratorTest {
+
+    @BeforeEach
+    @AfterEach
+    void resetUserIdGeneratorState() throws Exception {
+        Field field = UserIdGenerator.class.getDeclaredField("nextId");
```

Comment:
I feel like this is a non-standard way of doing things - I would suggest make a method reset in UserIdGenerator that resets the next id